src/arvgvfakecamera.c

```diff
@@ -564,6 +564,10 @@ _set_property (GObject *object, guint prop_id, const GValue *value, GParamSpec *
 	}
 }
 
+ArvFakeCamera* arv_gv_fake_camera_get_camera(ArvGvFakeCamera *gv_fake_camera)
+{return gv_fake_camera->priv->camera;}
```

Comment:
Coding style issue

Reply:
I guess you mean you want the braces on their own lines...

Reply:
Yes. The return value should be on its own line. Please also take care of the proper spacing.

Reply:
Trouble is that that would be fighting my editor, not to mention muscle memory. My editor needs to be set to 2 space indenting for other projects. If its so important, there are pretty formatting tools that can be used to enforce a particular spacing... Personally, I don't think it is too important.